boilerplate/fullstack/remix/app/routes/supertokens.sessioninfo.$.tsx

```diff
@@ -0,0 +1,15 @@
+import { json } from "@remix-run/node";
```

Comment:
make this api route `/sessoininfo` and not `/supertokens/sessioninfo` since we will use `/supertokens/*` for our middleware

Reply:
✔️